services/print/service/print_notification_service.go

```diff
@@ -0,0 +1,49 @@
+package service
+
+import (
+	"fmt"
+	"github.com/HackIllinois/api/services/print/config"
+	"github.com/HackIllinois/api/services/print/models"
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/session"
+	"github.com/aws/aws-sdk-go/service/sns"
+)
+
+var sess *session.Session
+var client *sns.SNS
+var QR_PREFIX = "hackillinois://qrcode/user?id=%d&identifier=%s"
+
+func init() {
+	sess = session.Must(session.NewSession(&aws.Config{
+		Region: aws.String(config.SNS_REGION),
+	}))
+	client = sns.New(sess)
+}
+
+/*
+	Returns the response from the SNS publish request
+*/
+func PublishPrintJob(job *models.PrintJob) (*sns.PublishOutput, error) {
+	identifier, err := GetUserInfo(job.ID)
+	if err != nil {
+		return nil, err
+	}
+	if !config.IS_PRODUCTION {
+		return &sns.PublishOutput{MessageId: aws.String("printjob-uuid")}, nil
+	}
+
+	payload := fmt.Sprintf(QR_PREFIX, job.ID, identifier)
```

Comment:
We want the checkin service to be the single service that controls the format of QR codes. You can `GET /checkin/qr/{id}/` for a specific user's qr code.

https://github.com/HackIllinois/api/blob/master/services/checkin/docs/Checkin.md#get-checkinqrid

Reply:
Yeah (I commented on this in another place in this PR), and if you want more information to be included in the QR code string, you could make a PR there.

Reply:
Does it make sense for checkin to handle the qr code? I think there's an argument to be made for it to be managed by whatever service is responsible for producing the actual labels.

Reply:
Technically, participants check-in first before a QR code is printed.

Reply:
That QR code will also be displayed in the mobile application and we will email it to them before checkin (incase their devices are too old to download the app). It's used in more places than just when it's being printed so I believe it fits better in the checkin service.